res/css/views/dialogs/_DeactivateAccountDialog.scss

```diff
@@ -21,3 +21,10 @@ limitations under the License.
 .mx_DeactivateAccountDialog .mx_DeactivateAccountDialog_input_section {
     margin-top: 60px;
 }
+
+.mx_DeactivateAccountDialog .mx_DeactivateAccountDialog_input_section input {
```

Comment:
I think I would indeed like to see this one become a `Field` as well. Is there some reason you didn't want to do that for this one?

Reply:
I tried to do as few changes but the label on the other field just irked me so much, I'll switch them both over, FIeld > input :D

Reply:
![image](https://user-images.githubusercontent.com/2403652/59562293-d9ddf280-9022-11e9-84d7-395970d5bfc8.png)